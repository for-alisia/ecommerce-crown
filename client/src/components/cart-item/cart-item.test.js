// @ts-nocheck
/** Libraries */
import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';

/** Components */
import CartItem from './cart-item.component';

const item = {
  name: 'Skirt',
  price: '25',
  imageUrl: 'http://someurl.com',
  quantity: 2,
};

it('expect to render a cart-item component', () => {
  const wrapper = shallow(<CartItem item={item} />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
