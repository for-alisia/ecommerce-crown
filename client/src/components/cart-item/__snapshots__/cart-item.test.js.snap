// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render a cart-item component 1`] = `
<styled.div>
  <styled.img
    alt="Skirt"
    src="http://someurl.com"
  />
  <styled.div>
    <styled.span>
      Skirt
    </styled.span>
    <styled.span>
      2
       x $
      25
    </styled.span>
  </styled.div>
</styled.div>
`;
